asghar/sprint1/infra/app.py

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env python3
+import os
+
+from aws_cdk import core as cdk
+
+# For consistency with TypeScript code, `cdk` is the preferred import name for
+# the CDK's core module.  The following line also imports it as `core` for use
+# with examples from the CDK Developer's Guide, which are in the process of
+# being updated to use `cdk`.  You may delete this import if you don't need it.
+from aws_cdk import core
+
+from infra.infra_stack import InfraStack
```

Comment:
Yes you are right. I first used different name for class but then we only have to differentiate stack so it can also be done by only defining stack id as a different name. Since class is only used to define stack infrastructure and when we cdk synth the name of stack is only decided by id that is given to stack class not class name.

But still if you say so I can change it